openrr-apps/src/bin/robot_command.rs

```diff
@@ -132,6 +135,18 @@ mod tests {
     }
 
     const SAMPLE_COMMAND_PATH: &str = "command/sample_cmd_urdf_viz.txt";
+    const COMMAND_PATHS: [&str; 10] = [
```

Comment:
Is there any reason to divide SAMPLE_COMMAND_PATH and COMMAND_PATHS?
How about pushing SAMPLE_COMMAND_PATH into COMMAND_PATHS if possible?

Reply:
I think pushing the path is better too. I updated it.